<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>733</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>19</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 225, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_internship">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1031</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>InternshipðŸ’»</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>Your name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_age">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_gender">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_email_ID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 218, 30);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="text">
     <string>Email ID</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_input_male">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Male</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_input_female">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Female</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_input_other">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>320</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Other</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_next_page">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 170, 255);
border-radius:20px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255);</string>
    </property>
    <property name="text">
     <string>Click to go to next page</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_name">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
padding:5px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Type your name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_age">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Type your age</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_email_ID">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>691</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
padding:5px;
border:2px solid rgba(0,0,0,0);
border-color:rgb(85, 85, 255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Type your email id</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
